serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,255 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
+
+It's recommended that you take a look at
+link:getting-familiar-with-our-tooling.html[this guide] to get familiar with our tooling and
+install the required dependencies.
+
+== Architecture
+
+In this guide, we'll create a Serverless Workflow application that serves a `hello_world` endpoint.
+The workflow comprises two states:
+
+. `Inject Hello World` - Injects a `Hello World` message into the response
+. `Inject Mantra` - Injects a `Mantra` message into the response
+
+image::getting-started/hello-world-workflow.png[]
+
+== Solution
+We recommend that you follow the instructions from <<boostrapping-the-project,Bootstrapping the Project>>
+and onwards to create the application step by step.
+
+However, you can go right to the completed example by cloning the git repository:
+
+The solution is located in the `kogito-quarkus-examples/serverless-workflow-hello-world`
+https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-hello-world[directory].
+
+[[boostrapping-the-project]]
+== Bootstrapping the Project
+
+Run the following command in a terminal window to create the project skeleton:
+
+.Using Quarkus CLI
+[source,shell]
+----
+quarkus create app \
+    -x=kogito-quarkus-serverless-workflow \
+    -x=quarkus-container-image-jib \
+    -x=quarkus-resteasy-jackson \
+    -x=quarkus-smallrye-openapi \
+    org.kie.kogito.examples:serverless-workflow-hello-world:1.0
+----
+
+.Using Maven
+[source,shell]
+----
+mvn io.quarkus.platform:quarkus-maven-plugin:2.9.1.Final:create \
+    -DprojectGroupId=org.kie.kogito.examples \
+    -DprojectArtifactId=serverless-workflow-hello-world \
+    -Dextensions="kogito-quarkus-serverless-workflow,quarkus-container-image-jib,quarkus-resteasy-jackson,quarkus-smallrye-openapi"
+cd getting-started
+----
+
+`org.kie.kogito.examples`, `serverless-workflow-hello-world` and `1.0` are respectively the group id, artifact id,
+and version of your project.
+
+This command will create a Maven Quarkus project in the `serverless-workflow-hello-world` directory with all the
+required Kogito dependencies.
+
+* `kogito-quarkus-serverless-workflow` - Adds support for Serverless Workflow
+* `quarkus-container-image-jib` - Adds support for Container Image Builds
+* `quarkus-resteasy-reactive-jackson` - Adds support for Reactive RESTEasy
+* `quarkus-smallrye-openapi` - Adds support for OpenAPI
```

Comment:
I will clarify that this dependecy is needed for support generation of Swagger documentation. Just Open API might lead to confusion, since SWF use OpenAPI spec to invoke remote REST servers, but the guide does not include any (thats good)